test/ipynb/mpl_tester.ipynb

```diff
@@ -2,24 +2,133 @@
  "cells": [
```

Comment:
There seems to be a lot of extra metadata added to this file which we probably don't want to merge in. Could you please commit a version of this file that hasn't been run and just includes the changes we want?

Reply:
@singhmeet11 This is a common problem when using the mpl_tester.ipynb on a local system. If you test with it, data from your local gets written back into it. What I do is copy the mpl_tester.ipynb from main to another name and place it outside of qiskit-terra on my local. Then I copy it into qiskit-terra/test/ipynb to do the testing, but before doing a commit, I delete the copy from there.

Reply:
Thanks for your suggestion I followed it and it mpl_tester.ipynb looks okay now